### DISCUSSION TITLE
How many aliases can we have per formula?

### DISCUSSION THREAD

**Discussion by andreaangiolillo** [Homebrew/discussions#3065] [Writing Formulae/Casks] [answered]
Hello,

I have recently discovered that formulas can have aliases. Now I am wondering if there is any limit on the number of aliases that we can add per formula.

Thanks!

**Comment by carlocab** [comment, marked as answer]
Homebrew itself does not impose a limit on the number of aliases a formula can have.

**Reply by andreaangiolillo** [reply]
Thanks! : )
